docs/installation.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Installation Guide
+nav_order: 2
+---
+
+In order to use the MarkLogic Connector in an Anypoint project, you must add the connector to the project.
+This can be accomplished in two different ways. You can either use the built-in assistant, which will find the connector
+in Anypoint Exchange and then add it to your project. Or, if you have the connector in a Maven repository, you can add
+the dependency to the project and Anypoint will retrieve the connector from the repository.
+
+No matter which method is used, once the MarkLogic Connector has been successfully added to the Anypoint project, "MarkLogic"
+will appear as a module in the Mule Palette.
+
+## Installing from the Anypoint Exchange
+
+1. Open Anypoint Exchange by clicking on the "Open Exchange" near the top-left of the window.
+2. If necessary, login with your Anypoint credentials.
+3. Search for "marklogic"
+4. Choose the "MarkLogic Connector - Mule 4" box.
+5. Click on "Add to project".
+6. When asked if you wish to proceed, click on "Proceed".
+
+## Installing manually with the JAR 
```

Comment:
I'm wondering about this part now if it strictly depends on a local Maven repository, which https://github.com/marklogic/marklogic-mule-connector/blob/master/doc/user-manual.adoc#to-install-this-connector-in-anypoint-studio-7 suggests it does. I can't imagine a normal Anypoint user ever following this approach - they would have needed to clone our repository and build the connector themselves. If they're savvy enough to do that, then I doubt they need to be told how to add it as a dependency. 

So I'm leaning towards only documenting how to add the connector via Anypoint Exchange - that seems like the normal user approach and what MuleSoft wants users to do. @wattsferry thoughts on this?